dashboard/src/components/ColoredCircle/ColoredCircle.tsx

```diff
@@ -40,13 +40,19 @@ const ColoredCircle = ({
   className,
   tooltipText,
 }: IColoredCircle): JSX.Element => {
+  const clearerCircles =
+    quantity == 0 &&
+    (tooltipText == 'global.failed' || tooltipText == 'global.inconclusive')
+      ? 'bg-opacity-20'
+      : '';
```

Comment:
Maybe we can put this behavior in every circle not just for `failed` or `inconclusive` IMO that will improve consistency

Reply:
If we can't, create a `fadeOnEmpty` prop for the component, as `tooltipText` is not a suitable value for this.

Reply:
I think the same, but ticket's title is clear.